app/views/users/new.html.erb

```diff
@@ -1,14 +1,16 @@
 <h1>New User</h1>
 
-<%
-=begin%>
- <%= render 'form', user: @user %> 
-<%
-=end%>
-
-<%
-=begin%>
- <%= link_to 'Back', users_path %> 
-<%
-=end%>
 
+<%= form_with(url: "/users", scope: "user", method: "post") do |form| %>
+
+<%= form.label(:username, "username") %>
+<%= form.text_field(:username) %>
```

Comment:
- [ ] You should have in this file the previous forms you worked on commented out. References, [1](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form_tag2.png), [2](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form2.png) and [3](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form1.png)